src/actions/connectionsActions.js

```diff
@@ -67,7 +67,8 @@ export const updateConnectionsAction = (theWalletId?: ?string = null) => {
     const removeContacts = [];
     const removeInvitations = [];
 
-    connections.forEach(connection => {
+    connections.forEach((connection: ApiContact) => {
```

Comment:
What do you think about returning from this function if no connections were received from the backend?
`if (isEmpty(connections)) return;`

I'm afraid that you could lose all your contacts if you have a shitty network connection.